src/lib/api.js

```diff
@@ -444,4 +444,8 @@ export default class RocketChatInstance {
       console.error(err.message);
     }
   }
+
+  getHost() {
```

Comment:
I think someone already got this function merged, please take a look. Rebase if necessary

Reply:
yeah got it thanks